effectiveOfficeBackend/src/main/resources/application.conf

```diff
@@ -5,11 +5,16 @@ ktor {
   }
 }
 database {
-    url = "jdbc:postgresql://localhost:15432/effectiveOfficeBackendDB"
-    username = "postgres"
-    password = "test1234567890"
+  url = "jdbc:postgresql://localhost:15432/effectiveOfficeBackendDB"
+  username = "postgres"
+  password = "test1234567890"
 }
 liquibase {
-    changelogFile = "changelog/changelog-master.yaml"
-    defaultSchemaName = "public"
+  changelogFile = "changelog/changelog-master.yaml"
+  defaultSchemaName = "public"
+}
+auth {
+  user {
+    emailDomain = "gmail.com"
+  }
```

Comment:
Move to .gitignore immediatly, all passwords in .env

Reply:
Done